src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        expects: (`
+            The function expects a single predicate function as its input.
+        `),
+        returns: (`
+            The function builds and returns a function returning true for
+            which inputs producing a falsey value when passed to the
+            inputted predicate and returning false for all inputs producing
+            a truthy value when passed to that predicate.
+        `),
+        examples: [
+            "basicUsage"
+        ],
+        related: [
+            "allPass", "anyPass", "nonePass"
+        ],
+    },
     attachSequence: false,
     async: false,
```

Comment:
At some point it will be desirable to have tests verify that the same function works called on its own, called async, called as a sequence method, and called as an async sequence method. Normal tests probably would exclusively test sync functions and use regular wrapped functions and sequence methods depending on whatever is more readable.